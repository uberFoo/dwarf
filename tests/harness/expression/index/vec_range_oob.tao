fn main() {
    let a = [1, 2, 3, 4];
    let b = a[2..5];
}
