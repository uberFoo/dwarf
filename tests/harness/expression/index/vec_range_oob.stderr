Interpreter exited with:
[31mError:[0m index out of bounds ðŸ’¥
   [38;5;246mâ•­[0m[38;5;246mâ”€[0m[38;5;246m[[0m vec_range_oob:3:16 [38;5;246m][0m
   [38;5;246mâ”‚[0m
 [38;5;246m3 â”‚[0m [38;5;249m [0m[38;5;249m [0m[38;5;249m [0m[38;5;249m [0m[38;5;249ml[0m[38;5;249me[0m[38;5;249mt[0m[38;5;249m [0m[38;5;249mb[0m[38;5;249m [0m[38;5;249m=[0m[38;5;249m [0m[38;5;249ma[0m[38;5;249m[[0m[38;5;249m2[0m[31m.[0m[31m.[0m[31m5[0m[38;5;249m][0m[38;5;249m;[0m
 [38;5;240m  â”‚[0m                [31mâ”€[0m[31mâ”¬[0m[31mâ”€[0m  
 [38;5;240m  â”‚[0m                 [31mâ•°[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m the index is [33m5[0m
 [38;5;240m  â”‚[0m 
 [38;5;240m  â”‚[0m [38;5;115mNote[0m: and the length of the array is [33m4[0m
[38;5;246mâ”€â”€â”€â•¯[0m

